FluentUI.Drawer/src/main/res/layout/view_persistent_sheet.xml

```diff
@@ -27,11 +28,14 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_marginTop="@dimen/fluentui_drawer_handle_vertical_margin"
-            android:layout_marginBottom="@dimen/fluentui_drawer_handle_vertical_margin"
-            android:importantForAccessibility="no"
+            android:paddingTop="@dimen/fluentui_drawer_handle_vertical_margin"
+            android:paddingBottom="@dimen/fluentui_drawer_handle_vertical_margin"
+            android:paddingStart="@dimen/fluentui_drawer_handle_horizontal_margin"
+            android:contentDescription=" "
```

Comment:
Can we check for accessibility here once? earlier it was market as not important and now content description is empty

Reply:
An empty description makes an expansion call by default.